source/user-guide/publishing/managing-privileges.rst

```diff
@@ -68,13 +68,27 @@ Editing
 
 An *administrator* can edit any metadata.
 
-A *reviewer* can edit a metadata if:
+A *reviewer* or an *Editor* can edit a metadata if:
 
-* The metadata owner is member of one of the groups assigned to the reviewer.
+* All metadata that has the edit privilege selected for one of the groups they are a member of.
```

Comment:
I find the following statement is a little confusing - it is possible to clarify this statement?

It states "can edit **a** metadata if" followed by "**All** metadata".  It first mentioned being able to edit a single metadata record but then says that this can be done based on multiple metadata.   Is this related to one metadata or multiple metadata?

I also think that "that has the edit privilege selected" can be replaced by "has 'edit' privilege"

Here is a suggested replacement - but I'm not sure why "The" was changed to "All" so this may not be correct.

"The metadata has 'edit' privilege for one of the groups they are a member of"